index.html

```diff
@@ -8,10 +8,30 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <nav>
-        <ul>
-            <li>Logo</li>
-        </ul>
-    </nav>
+    <div id="header-background">
+        <nav id="flex-nav">
+            <ul>
+                <li id="logo"><a href="index.html">My Logo</a></li>
+                <li id="burguer-menu-icon ">
+                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                </li>
+            </ul>
+        </nav>
+        <div id="primary-text">
+            <h2 id="primary-text-header">I'm Enoque <br/> Glad to see<br/>you!</h2>
+            <p id="hello-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p id="connect">LET'S CONNECT</p>
+            <div id="social">
+                <img src="./img/twitter.png" alt="twitter link">
+                <img src="./img/Linkedin.png" alt="Linkedin link">
+                <img src="./img/m.png" alt="m link">
+                <img src="./img/github.png" alt="twitter link">
+                <img src="./img/m2.png" alt="m2 link">
+            </div>
+        </div>
+    </div>
+    <img src="./img/Rectangle 6.svg" alt="Rectangle" id="rectangle-6">
+    <img src="./img/Ellipse 3.svg" alt="ellipse" id="ellipse-3">
+    <img src="./img/Rectangle 4.svg" alt="rectangle" id="rectangle-4">
 </body>
```

Comment:
I can see you used flexbox but only applied it to the logo and the hamburger menu, please try as much as possible to use flexbox to align all the contents, since it gives you a lot of options to align all your elements and also helps to make your project more responsive.